fakenet.js

```diff
@@ -0,0 +1,58 @@
+const Fakenet = require('./fixtures/FakeNet/FakeNet');
+const { Wallet } = require('./src/index');
+
+const wallet = new Wallet({
+  // mnemonic: 'during develop before curtain hazard rare job language become verb message travel',
```

Comment:
we need both?

Reply:
Those are the two mnemonic used with the fakenet data set. 
If we removes it, we need a place to store it, because later, we will need to have them init on our test-ci with a .getTransactionHistory() expect test.

Reply:
File removed. Mnemonic store in a json file in fixtures/wallet/mnemonic.json